CC Processing

juggernautsei opened the pull request:
Building credit card processing into the program. The overall goal is to have Stripe.com, Authorize.net and PayPal.com available for processing of payments and recording those payments in the EMR for use by the billing staff. 

Have spoken with a physician about how the workflow would work for him. Currently building out the process the way he proposed. The working system can be reviewed at 
https://omp.openmedpractice.com/ui-demo login credentials are on the landing page.
The heart of the code is located in the /interface/payment_process/stripe
Stripe is the only completely functional code at this time. 
The goal is to have this code included in the next release.

bradymiller commented on interface/main/left_nav.php:
```diff
@@ -1530,7 +1530,6 @@ function selpopup(selobj) {
         <ul>
           <?php if (!$GLOBALS['disable_calendar']) genMiscLink('RTop','rep','0',xl('Appointments'),'reports/appointments_report.php'); ?>
-          <?php if (!$GLOBALS['disable_pat_trkr'] && !$GLOBALS['disable_calendar']) genMiscLink('RTop','rep','0',xl('Patient Flow Board'),'reports/patient_flow_board_report.php'); ?>          
           <?php  genMiscLink('RTop','rep','0',xl('Encounters'),'reports/encounters_report.php'); ?>
```
You have removed others code/feature. This is a git issue.

bradymiller commented:
I'd recommend focusing on Terry's comments and figuring out how not to remove other's code in git/github for the next revision. Let us know if need pointers on git/github to do this. I'd also suggest keeping your master branch a mirror of the official codebase and then working on a separate git branch.

juggernautsei commented:
I have updated the 4_2_0-to-4_2_1_upgrade.sql file and added some form validations. 
Unfortunately trying to get the github right I wiped out the last change log. I'll keep at it till it get it right. I have created a new branch for the credit card gateway. I will be moving the updates to it once I can get it populated.
